src/backfill/looper/looper.ts

```diff
@@ -1,143 +0,0 @@
-import Logger from "bunyan";
```

Comment:
Sad to see this code go as it took care of quite some issues with ratelimiting, retrying, etc. But I never pushed for adopting this backfilling style. cc @joshkay10 is there anything we still need to fix in the backfilling mechanism to make it more stable? We can still use this for inspiration

Reply:
We don't have the preemptive ratelimiting smarts yet.... I still want to spike this solution and bring over a couple of the recent enhancements, it does feel like a shame to drop it without an honest go. The best backfill solution will be an ever going debate, but it's worth evaluating the effort vs reward.